src/test/java/com/microsoft/sqlserver/jdbc/clientcertauth/ClientCertificateAuthenticationTest.java

```diff
@@ -26,8 +26,7 @@
 
 
 /**
- * Tests client certificate authentication feature
- * The feature is only supported against SQL Server Linux CU2 or higher.
+ * Tests client certificate authentication feature The feature is only supported against SQL Server Linux CU2 or higher.

```

Comment:
Missing punctuation. Should be either:

- Tests client certificate authentication feature. The feature is only supported against SQL Server Linux CU2 or higher. OR
- Tests client certificate authentication feature; the feature is only supported against SQL Server Linux CU2 or higher.